packages/flame_isolate/example/lib/units/worker.dart

```diff
@@ -0,0 +1,153 @@
+import 'package:flame/components.dart';
+import 'package:flame/flame.dart';
+import 'package:flutter/material.dart' hide Animation;
+import 'package:flutter_isolates_example/colonists_game.dart';
+import 'package:flutter_isolates_example/constants.dart';
+import 'package:flutter_isolates_example/objects/colonists_object.dart';
+import 'package:flutter_isolates_example/standard/int_vector2.dart';
+import 'package:flutter_isolates_example/standard/pair.dart';
+import 'package:flutter_isolates_example/units/actions/movable.dart';
+
+class Worker extends PositionComponent
```

Comment:
Use a `SpriteAnimationGroupComponent` instead (you are basically re-implementing the logic of that component in here)